src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -151,40 +151,44 @@ const personalDetailsSelector = (personalDetails) =>
  * use it to prevent re-renders from parent re-renders.
  */
-export default React.memo(
-    compose(
-        withCurrentReportID,
-        withReportCommentDrafts({
-            propName: 'comment',
-            transformValue: (drafts, props) => {
-                const draftKey = `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${props.reportID}`;
-                return lodashGet(drafts, draftKey, '');
-            },
-        }),
-        withOnyx({
-            fullReport: {
-                key: (props) => ONYXKEYS.COLLECTION.REPORT + props.reportID,
-            },
-            reportActions: {
-                key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
-                canEvict: false,
-            },
-            personalDetails: {
-                key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-                selector: personalDetailsSelector,
-            },
-            preferredLocale: {
-                key: ONYXKEYS.NVP_PREFERRED_LOCALE,
-            },
-            policies: {
-                key: ONYXKEYS.COLLECTION.POLICY,
-            },
-        }),
-        withOnyx({
-            parentReportActions: {
-                key: ({fullReport}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${fullReport.parentReportID}`,
-                canEvict: false,
-            },
-        }),
-    )(OptionRowLHNData),
+export default compose(
+    withCurrentReportID,
+    withReportCommentDrafts({
+        propName: 'comment',
+        transformValue: (drafts, props) => {
+            const draftKey = `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${props.reportID}`;
+            return lodashGet(drafts, draftKey, '');
+        },
+    }),
+    withOnyx({
+        fullReport: {
+            key: (props) => ONYXKEYS.COLLECTION.REPORT + props.reportID,
+        },
+        reportActions: {
+            key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
+            canEvict: false,
+        },
+        personalDetails: {
+            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
+            selector: personalDetailsSelector,
+        },
+        preferredLocale: {
+            key: ONYXKEYS.NVP_PREFERRED_LOCALE,
+        },
+        policies: {
+            key: ONYXKEYS.COLLECTION.POLICY,
+        },
+    }),
+    withOnyx({
+        parentReportActions: {
+            key: ({fullReport}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${fullReport.parentReportID}`,
+            canEvict: false,
+        },
+    }),
+)(
+    memo(OptionRowLHNData, (prevProps, nextProps) => {
+        const prevParentReportActions = prevProps.parentReportActions[prevProps.fullReport.parentReportActionID];
+        const nextParentReportActions = nextProps.parentReportActions[nextProps.fullReport.parentReportActionID];
+        return prevParentReportActions === nextParentReportActions && _.isEqual(_.omit(prevProps, 'parentReportActions'), _.omit(nextProps, 'parentReportActions'));
```

Comment:
For readability I think it would better to write this as

```js
if (prevParentReportActions !== nextParentReportActions) {
    return false;
}
return _.isEqual(_.omit(prevProps, 'parentReportActions'), _.omit(nextProps, 'parentReportActions'));
```